ts/Extensions/Annotations/NavigationBindingsDefaults.ts

```diff
@@ -148,6 +148,12 @@ const navigation: NavigationOptions = {
         circleAnnotation: {
             /** @ignore-option */
             className: 'highcharts-circle-annotation',
+            /**
+             * @type      {Highcharts.AnnotationsOptions}
+             * @extends   navigation.annotationsOptions
+             * @apioption navigation.bindings.circleAnnotation.annotationsOptions
+             */
+            annotationsOptions: {},
```

Comment:
If you define the `@apioption` above, then you don't have to define empty `annotationsOptions` object, right? Could you double-check it?

Reply:
Yes it still works! Thanks!